modules/ipi-install-configuring-host-network-interfaces-in-the-install-config.yaml-file.adoc

```diff
@@ -2,62 +2,119 @@
 //
 // installing_bare_metal_ipi/ipi-install-installation-workflow.adoc
 
-[id="configuring-host-network-interfaces-in-the-install-config-yaml-file_{context}"]
-= (Optional) Configuring host network interfaces in the `install-config.yaml` file
+:_content-type: PROCEDURE
+[id="configuring-host-network-interfaces-in-the-install-config.yaml-file_{context}"]
+= Optional: Configuring host network interfaces in the `install-config.yaml` file
```

Comment:
Avoid backticks in section headings ([repo guideline](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assemblymodule-titles-and-section-headings)), though it looks like in this case this would also affect a number of existing section titles throughout the assembly. :shrug: